cli/pcluster/config/validators.py

```diff
@@ -1253,3 +1253,31 @@ def fsx_ignored_parameters_validator(section_key, section_label, pcluster_config
 def _describe_ec2_key_pair(key_pair_name):
     """Return information about the provided ec2 key pair."""
     return boto3.client("ec2").describe_key_pairs(KeyNames=[key_pair_name])
+
+
+def ebs_volume_iops_validator(section_key, section_label, pcluster_config):
+    errors = []
+    warnings = []
+
+    section = pcluster_config.get_section(section_key, section_label)
+    volume_size = section.get_param_value("volume_size")
+    volume_type = section.get_param_value("volume_type")
+    volume_type_to_iops_ratio = {"io1": 50, "io2": 500}
+    volume_iops = section.get_param_value("volume_iops")
+    min_iops = 100
+    max_iops = 64000
+    if volume_type in volume_type_to_iops_ratio:
+        if volume_iops and (volume_iops < min_iops or volume_iops > max_iops):
+            errors.append(
+                "IOPS rate must be between {min_iops} to {max_iops} when provisioning {volume_type} volumes.".format(
```

Comment:
sorry if my previous comment had a typo in it: this should be "{min_iops} and {max_iops}" instead of "{min_iops} to {max_iops}"

Reply:
Done!